### ISSUE TITLE
missing version tags

### ISSUE THREAD

**Issue by Fabricio872** [approvals/ApprovalTests.php#6]
Hello, when I was installing this package I got a message: "Could not find a version of package approvals/approval-tests matching your minimum-stability (stable). Require it with an explicit version constraint allowing its desired stability."

**Comment by franiglesias** [comment]
For that situation you can use:

```
composer req --dev approvals/approval-tests dev-Main
```

**Comment by floriankraemer** [comment]
Why no version tags?

**Comment by gquemener** [comment]
Yes, please. Could you make a tag to ease installation on project with "minimum-stability" set as "stable"? I can take care of it, as I am interested in becoming a maintainer as well.
